DER/DerivativesContracts/DerivativesBasics.rdf

```diff
@@ -232,6 +236,50 @@ See https://opensource.org/licenses/MIT.</dct:license>
 		<cmns-av:explanatoryNote>Substitution may be done through contract novation, for example. A derivatives clearing organization (DCO) also arranges or provides, on a multilateral basis, for the settlement or netting of obligations, or otherwise provides clearing services or arrangements that mutualize or transfer credit risk among participants.</cmns-av:explanatoryNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-drc-bsc;EconomicRateBasedDerivative">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-bsc;RateBasedDerivative"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-fi-fi;hasUnderlier"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&cmns-rlcmp;isPlayedBy"/>
+						<owl:someValuesFrom rdf:resource="&fibo-ind-ei-ei;EconomicIndicator"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>economic rate-based derivative</rdfs:label>
+		<skos:definition>rate-based derivative whose underlier is some economic indicator</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-drc-bsc;ForwardRateAgreement">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-bsc;InterestRateDerivative"/>
+		<rdfs:label xml:lang="en">forward rate agreement</rdfs:label>
+		<skos:definition xml:lang="en">agreement to exchange an interest rate commitment on a notional amount</skos:definition>
+		<cmns-av:abbreviation xml:lang="en">FRA</cmns-av:abbreviation>
```

Comment:
need to add to the definition "at a future date"

Reply:
Done.